bq-connector/docai_bq_connector/connector/BqDocumentMapper.py

```diff
@@ -109,6 +109,10 @@ def _parse_entities(self, entities) -> DocumentRow:
                     )
                 )
             else:
+                content = entity.mention_text
```

Comment:
Nice catch, it looks like the block from 86-89 can be moved outside of the if block, and eliminate the need for duplicative value setting here.

Reply:
Indeed!